src/http_request.rs

```diff
@@ -37,27 +41,41 @@ impl HttpRequest {
         // If the request is complete we are returning the response in the stream
         if request_status.is_complete() {
             info!("Request is complete.");
-            match request.path {
-                Some(ref _path) => {
-                    info!("Request path is: {}", _path);
-                    let response = "HTTP/1.1 200 OK\r\n\r\n";
-                    stream.write_all(response.as_bytes()).unwrap();
-                    stream.flush().unwrap();
-                }
-                None => {
-                    info!("Request path is missing.");
-                }
-            }
+            let path = request.path.map(String::from);
+            let method = match request.method.unwrap() {
```

Comment:
unwrap ? :eye:  XD

Reply:
This is a part of weird things I told in the previous conversation. We will validate the architecture, and I will refactor and remove that weird things ^^